ExSwiftTests/StringExtensionsTests.swift

```diff
@@ -10,50 +10,50 @@ import Quick
 import Nimble
 
 class StringExtensionsSpec: QuickSpec {
-    
+
     override func spec() {
-    
+
         /**
         *  String.length
         */
         it("length") {
-            
+
             expect("".length) == 0
             expect("A".length) == 1
             expect("😱".length) == 1
             expect("∞".length) == 1
             expect("∞aA".length) == 3
-            
+
         }
-        
+
         /**
         *  String[x]
         */
         it("subscript") {
-            
+
             let string = "∆Test😗"
-            
+
             expect(string[0]) == "∆"
             expect(string[1]) == "T"
-            
+
             expect(string[string.length - 1]) == "😗"
             expect(string[1..<6]) == "Test😗"
-            
+
             if let s = "hello"[0] {
                 expect(s) == "h"
             } else {
                 fail("string[0]")
             }
-            
+
             expect("ABCD"[0, 2]) == ["A", "C"]
-            
+
         }
-        
+
```

Comment:
There is a lot of trailing spaces in file, so I remove them also. ( According general code style guidelines http://programmers.stackexchange.com/a/121560 )